### PR TITLE
Added "The Lurking Fear", "The Silver Key", "The Haunter of the Dark", and "The Thing on the Doorstep"

### PR DESCRIPTION
**Pull request by HendrikBK**

### CODE REVIEW

File: src/epub/text/pickmans-model.xhtml [hunk 1]
```diff
@@ -46,7 +46,7 @@
...
+			<p>It was a colossal and nameless blasphemy with glaring red eyes, and it held in bony claws a thing that had been a man, gnawing at the head as a child nibbles at a stick of candy. Its position was a kind of crouch, and as one looked one felt that at any moment it might drop its present prey and seek a juicier morsel. But, damn it all, it wasn’t even the fiendish subject that made it such an immortal fountainhead of all panic⁠—not that, nor the dog face with its pointed ears, bloodshot eyes, flat nose, and drooling lips. It wasn’t the scaly claws nor the mold-caked body nor the half-hooved feet⁠—none of these, though anyone of them might well have driven an excitable man to madness.</p>
```
Comment (acabal) [inline src/epub/text/pickmans-model.xhtml:49]: This is an incorrect change

**Review by acabal — CHANGES_REQUESTED** [review]
We also need to add `<dc:source>` with PG transcriptions and page scans for these. Did you also add them for the other items you recently added? I can't recall.

File: src/epub/text/the-thing-on-the-doorstep.xhtml [hunk 2]
```diff
@@ -14,15 +14,15 @@
...
+				<p>By the time he was twenty-five Edward Derby was a prodigiously learned man and a fairly well-known poet and <em>fantasist</em>, though his lack of contacts and responsibilities had slowed down his literary growth by making his products derivative and over-bookish. I was perhaps his closest friend⁠—finding him an inexhaustible mine of vital theoretical topics, while he relied on me for advice in whatever matters he did not wish to refer to his parents. He remained single⁠—more through shyness, inertia and parental protectiveness than through inclination⁠—and moved in society only to the slightest and most perfunctory extent. When the war came both health and ingrained timidity kept him at home. I went to Plattsburgh for a commission, but never got overseas.</p>
```
Comment (acabal) [inline src/epub/text/the-thing-on-the-doorstep.xhtml:25]: This is incorrect, `fantaisiste` is a French word, not English emphasis. This should be reverted and changed to `<i xml:lang="fr">`

File: src/epub/text/the-thing-on-the-doorstep.xhtml [hunk 3]
```diff
@@ -94,7 +94,7 @@
...
-				<p>“I had to save myself⁠—I had to, Dan! She’d have got me for good at Hallowmass⁠—they hold a Sabbat up there beyond Chesuncook, and the sacrifice would have clinched things. She’d have got me for good⁠—she’d have been I, and I’d have been she⁠—forever⁠—too late⁠—My body’d have been hers for good⁠—She’d have been a man, and fully human, just as she wanted to be⁠—I suppose she’d have put me out of the way⁠—killed her own ex-body with me in it, damn her, <em>just as she did before</em>⁠—just as she, he, or it did before⁠—”</p>
+				<p>“I had to save myself⁠—I had to, Dan! She’d have got me for good at Hallowmass⁠—they hold a Sabbath up there beyond Chesuncook, and the sacrifice would have clinched things. She’d have got me for good⁠—she’d have been I, and I’d have been she⁠—forever⁠—too late⁠—My body’d have been hers for good⁠—She’d have been a man, and fully human, just as she wanted to be⁠—I suppose she’d have put me out of the way⁠—killed her own ex-body with me in it, damn her, <em>just as she did before</em>⁠—just as she, he, or it did before⁠—”</p>
```
Comment (acabal) [inline src/epub/text/the-thing-on-the-doorstep.xhtml:93]: Incorrect change, `Sabbat` means a witch's festival, `Sabbath` means a day of rest and worship

**Review by acabal — CHANGES_REQUESTED** [review]

**Comment by HendrikBK** [comment]
The change in "Pickman's Model" was an accident, I just reverted.
I did add source in metadata for the other stories, I just forgot about these ones, they're present now.
I don't understand the issue with "any one" in "The Haunter of the Dark", there doesn't seem to be any change of the kind the section, perhaps you meant another wish?

**Comment by acabal** [comment]
> The change in "Pickman's Model" was an accident, I just reverted. I did add source in metadata for the other stories, I just forgot about these ones, they're present now. I don't understand the issue with "any one" in "The Haunter of the Dark", there doesn't seem to be any change of the kind the section, perhaps you meant another wish?

`gases born of long decay⁠—anyone of numberless` is in correct

**Comment by acabal** [comment]
Good work, thanks!

**Pull request merged**